qiskit_experiments/library/characterization/__init__.py

```diff
@@ -32,6 +33,7 @@
 
 .. autosummary::
     :toctree: ../stubs/
+    :template: autosummary/experiment.rst
```

Comment:
Oh I found why it caused an error. This is a wrong template. 
https://github.com/Qiskit/qiskit-experiments/blob/36fe3134e4f1ae9e3e2b5654d7169905ac191365/qiskit_experiments/library/randomized_benchmarking/__init__.py#L35